dev/AppNotifications/AppNotificationUtility.h

```diff
@@ -62,4 +62,6 @@ namespace Microsoft::Windows::AppNotifications::Helpers
     winrt::Microsoft::Windows::AppNotifications::AppNotification ToastNotificationFromToastProperties(ABI::Microsoft::Internal::ToastNotifications::INotificationProperties* properties);
 
```

Comment:
github won't let me comment on the lines for `ConvertUtf8StringToWideString` so here you go

Are you aware of `dev\common\Microsoft.Utf8.h`?

`ConvertUtf8StringToWideString` can be replaced with a `ToUtf16()` in there. There's functions to give you a `std::unique_ptr<WCHAR[]>`, `std::wstring()` or `winrt::hstring`.

No `wil::unique_cotaskmem_string` overload but if you really need that it's not hard to add

Reply:
Added to this bug -> #2307